apis/scheduledtask.k8ssandra.io/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the scheduledtask.k8ssandra.io v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=scheduledtask.k8ssandra.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "scheduledtask.k8ssandra.io", Version: "v1alpha1"}
```

Comment:
Is there a reason you didn't just add this to the control.k8ssandra.io group and the same api pkg? Why do we need a new group?

Reply:
I don't see a lot of difference either way, I can move it into the existing group if you have feelings about it.